Firestore/core/src/firebase/firestore/api/firestore.h

```diff
@@ -40,11 +40,13 @@ NS_ASSUME_NONNULL_BEGIN
 
 OBJC_CLASS(FIRQuery);
 OBJC_CLASS(FIRTransaction);
-OBJC_CLASS(FSTFirestoreClient);
 OBJC_CLASS(NSString);
 
 namespace firebase {
 namespace firestore {
+namespace core {
```

Comment:
Nit: you can just `include` the header. In general, the style guide recommends preferring includes to forward declarations. It was forward-declared previously to avoid importing an Objective-C header.

Reply:
This would break cc_compilation_test as `firestore_client.h` is only meant for Obj-C++ at the moment.